backend/prisma/schema.prisma

```diff
@@ -42,4 +42,13 @@ model Chapter {
   content   String
   courseId  Int
   course    Course  @relation(fields: [courseId], references: [id])
+  lectures  Lecture[]
+}
+
+model Lecture {
```

Comment:
Maybe it would be easier to query the lectures by their user ID when checking if a user has access to a lecture?